src/faction.cpp

```diff
@@ -0,0 +1,48 @@
+#include "faction.h"
+#include "lib/netplay/netplay.h"
+
+const struct FACTION factions[NUM_FACTIONS] = {
+	{ "Normal",
+		{
+		}
+	},
+	{
+		"NEXUS",
+		{
+			{"blwallc1.pie", "blwallc3.pie"},
+			{"blwallh.pie", "blwall3.pie"},
+			{"blhq.pie", "blhq3.pie"},
+			{"blguard1.pie", "blguard3.pie"},
+			{"blguardr.pie", "blgrdnex.pie"},
+		}
+	},
+	{
+		"Collective",
+		{
+			{"blwallc1.pie", "blwallc2.pie"},
+			{"blwallh.pie", "blwall2.pie"},
+			{"blhq.pie", "blhq4.pie"},
+			{"blguard1.pie", "blguard2.pie"},
+			{"blguardr.pie", "blguardn.pie"},
+		}
+	}
+};
+
+iIMDShape* getFactionIMD(uint8_t player, iIMDShape* imd)
```

Comment:
Ok then we would need a function getPlayerFaction or so, idk if i added one already ... i had a break of a month now because of work. I'll change that

Reply:
@Kailari i changed it now that you have a function `const FACTION* getPlayerFaction(uint8_t player)`
[you can see it here](https://github.com/cpdef/warzone2100/blob/factions_test_2/src/faction.h#L33#L42)
is it ok like that? Then i'll resolve the conversation

Reply:
Definitely better than before, nice! 👍